libs/map-editor/src/GameMap/GameMapAreas.ts

```diff
@@ -44,7 +46,14 @@ export class GameMapAreas {
                     console.warn(`Area name "${name}" must be a rectangle`);
                     return;
                 }
-                this.staticAreas.push(area as ITiledMapRectangleObject);
+                for (let property of properties){
+                    if(property.name === 'dynamic' && property.type === "bool" && property.value){
+                        this.dynamicAreas.push(area as ITiledMapRectangleObject);
+                    }else{
+                        this.staticAreas.push(area as ITiledMapRectangleObject);
+                    }
```

Comment:
This should be put outside the for loop. Otherwise, you are adding a staticArea for every property that is not "dynamic".